binplace.targets

```diff
@@ -50,7 +50,10 @@
     <BinPlaceConfiguration Condition="'$(BinPlaceILCInputFolder)' == 'true'" Include="uapaot-$(_bc_OSGroup)">
       <RuntimePath>$(ILCFXInputFolder)</RuntimePath>
     </BinPlaceConfiguration>
-
+    <!-- And the UAP folder for the F5 (CoreCLR UAP-debugging) scenario -->
+    <BinPlaceConfiguration Condition="'$(BinPlaceTestSharedFramework)' == 'true'" Include="uap-$(OSGroup)">
```

Comment:
I thought we said that the Runtime Dir had everything we wanted so that we didn't need to binplace these guys into a separate directory